model/transformers/dffml_model_transformers/qa/qa_accuracy.py

```diff
@@ -0,0 +1,71 @@
+import os
+from typing import AsyncIterator, List
+
+from dffml.base import config
+from dffml.record import Record
+from dffml.feature import Feature
+from dffml.model import ModelNotTrained
+from dffml.util.entrypoint import entrypoint
+from dffml.accuracy import (
+    AccuracyScorer,
+    AccuracyContext,
+    InvalidNumberOfFeaturesError,
+)
+
+from .qa_model import logger
+
+
+@config
+class QAAccuracyConfig:
+    pass
+
+
+class QAAccuracyContext(AccuracyContext):
+    async def score(
+        self, records: AsyncIterator[Record], features: List[Feature],
+    ):
+        if not os.path.isfile(
+            os.path.join(self.parent.config.output_dir, "pytorch_model.bin")
```

Comment:
I'm realizing we don't have access to any of the model parameters. Sorry, we shouldn't have taken out these methods. I'm now remembering that's why we modified the accuracy method in the base model class. So that we could override it if necessary. We should revert these last six commits, and modify the signature of the accuracy method to match the new version that takes the scorer. However we should use `logger.warning` to mention that we aren't going to use the accuracy scorer provided